tests/aatriangle_test.cpp

```diff
@@ -0,0 +1,30 @@
+//  SuperTux
+//  Copyright (C) 2016 Tapesh Mandal <tapesh.mandal@gmail.com>
+//
+//  This program is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#include <gtest/gtest.h>
+
+#include "math/aatriangle.hpp"
+
+TEST(AATriangleTest, aatriangle_test)
```

Comment:
I think that rather than `aatriangle_test`, this test should be called `vertical_flip_test` as it tests specifically that function. (Keep the `AATriangleTest` though)